assets/intro.js

```diff
@@ -0,0 +1,89 @@
+(function ($) {
+	var options = importUploadSettings;
+	var uploader, itemTemplate, statusTemplate, errorTemplate;
+
+	// progress and success handlers for media multi uploads
+	var items = {};
```

Comment:
The ones at the top are declared there to hoist their scope up from the individual functions to the container, I prefer inline `var` as a matter of style though.